LEGO1/lego/legoomni/src/common/mxcontrolpresenter.cpp

```diff
@@ -78,7 +78,10 @@ void MxControlPresenter::EndAction()
 MxBool MxControlPresenter::FUN_10044270(MxS32 p_x, MxS32 p_y, MxPresenter* p_presenter)
 {
 	assert(p_presenter);
-	MxStillPresenter* presenter = (MxStillPresenter*) p_presenter;
+	MxStillPresenter* presenter = dynamic_cast<MxStillPresenter*>(p_presenter);
+	if (!presenter) {
+		return FALSE;
```

Comment:
Does this happen often?
As this changes behavior, should we log a message here?

Reply:
Sounds good - I don't think that should happen at all